src/components/ui/ShippingSimulation/shipping-simulation.module.scss

```
@@ -63,13 +47,16 @@
     display: flex;
     align-items: center;
     padding-top: var(--fs-shipping-simulation-link-padding-top);
-    padding-bottom: var(--fs-shipping-simulation-link-padding-bottom);
 
     svg {
       margin-left: var(--fs-spacing-0);
     }
   }
 
+  [data-fs-shipping-simulation-header] {
+    padding-top: var(--fs-shipping-simulation-header-padding-bottom);
```

```suggestion
    padding-top: var(--fs-shipping-simulation-header-padding-top);
```
padding-top or padding-bottom? ahaha Didn't find any padding-bottom use here.